cmd/platform.go

```diff
@@ -283,6 +286,35 @@ func createGiteaClient(flag *flag.FlagSet, verifyFlags bool) (multigitter.Versio
 	return vc, nil
 }
 
+// TODO: Add more code to the client
+func createBitbucketCloudClient(flag *flag.FlagSet, verifyFlags bool) (multigitter.VersionController, error) {
+	workspaces, _ := flag.GetStringSlice("org")
+	users, _ := flag.GetStringSlice("user")
+	repos, _ := flag.GetStringSlice("repo")
+	username, _ := flag.GetString("username")
+	sshAuth, _ := flag.GetBool("ssh-auth")
+
+	if verifyFlags && len(workspaces) == 0 && len(users) == 0 && len(repos) == 0 {
+		return nil, errors.New("no workspace, user or repository set")
+	}
+
+	if username == "" {
```

Comment:
We initially tried this but the problem is most OAuth functionality is gated behind bitbucket premium, app passwords are what is most accessible for bitbucket users and are going to support it first before we add OAuth support since many bitbucket users are not premium.

The problem is the amount of permissions multi-gitter needs, in order to access multiple projects or workspaces you need a higher permission token and repository tokens(that are free) are far too limited to handle updating multiple repositories in a project or workspace.

Reply:
App passwords are what is recommended over OAuth tokens for non-premium users, they have some similar features including the ability to limit access to what an app password can do and the ability to revoke and re-generate easily.

We would like to add OAuth support as an additional feature in a separate PR to keep the scope of this change small as well.

Reply:
SGTM 👍